requirements.txt

```diff
@@ -1,3 +1,4 @@
+beaker-client==27.4
```

Comment:
It would be nice to improve the commit message to include some reasoning about your decisions. (e.g. why setting the the version)

Reply:
I tested this with versions 27.4 and 28 in requirements i will leave >=27